rolling-shutter/cmd/shutterservice/shutterservice.go

```diff
@@ -0,0 +1,53 @@
+package shutterservice
```

Comment:
The other keyper implementations all have `keyper` in their command name. I think we should keep that convention.